0-ad-history-mod-package/simulation/templates/structures/athen/prytaneion.xml

```diff
@@ -25,6 +25,10 @@
     <Classes datatype="tokens">ConquestCritical CivSpecific</Classes>
     <VisibleClasses datatype="tokens">Council</VisibleClasses>
     <Icon>structures/tholos.png</Icon>
+    <History>
+        The Prytaneion was the official residence of the city's or city-state's executives in Ancient Greece. It was usually located in the city's center, near or at the Agora, the public marketplace and forum. Because Prytaneia were constructed a bit differently from city to city, today it can be difficult to identify them. But, they essentially worked as a town hall, in which not only official political but also religious ceremonies and celebrations were held.
```

Comment:
```suggestion
        The Prytaneion was the official residence of the city's or city-state's executives in Ancient Greece. It was usually located in the city's center, near or at the Agora, the public marketplace and forum. Because Prytaneia were constructed a bit differently in each city, today it can be difficult to identify them. However, their uses were the same. The Prytaneion was essentially the town hall, in which not only official political but also religious ceremonies and celebrations were held.
```